recipes-extended/sqlitecpp/sqlitecpp_3.3.1.bb

```diff
@@ -0,0 +1,20 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=3d149a8cb6c7dab01d788f477294cebc"
+
+SRC_URI = "git://@github.com/SRombauts/SQLiteCpp.git;branch=master;protocol=https"
```

Comment:
Is the `@` in `git://@github.com/` intentional?

Reply:
That was a typo, removed